packages/runtime/src/next/index.tsx

```diff
@@ -115,7 +115,9 @@ export async function getServerSideProps(
     apiOrigin: getApiOrigin(),
   })
   const path = '/' + (ctx.params?.path ?? []).join('/')
-  const snapshot = await makeswift.getPageSnapshot(path, { preview: true })
+  const snapshot = await makeswift.getPageSnapshot(path, {
+    siteVersion: Makeswift.getSiteVersion(ctx.previewData),
+  })
```

Comment:
This needs to be hardcoded to `MakeswiftSiteVersion.Working` to preserve its behavior.